sklearn/neighbors/_quad_tree.pxd

```diff
@@ -4,12 +4,12 @@
 # See quad_tree.pyx for details.
 
 import numpy as np
```

Comment:
I don't think that we need this import anymore.

Reply:
Well spotted, thanks @glemaitre